### ISSUE TITLE
[Midnight] No Rifts to exit Midnight Dimension once abducted

### ISSUE THREAD

**Issue by Vaygrim** [JoelZBub/organic-free-range-pack#63]
### How Can We Reproduce the Bug:
1. Get abducted by Rifters

### Please complete the following information:
 - Modpack Version Number : 1.1.1
 - Is OptiFine installed? : NO
 - Does this issue affect Server, Client, or Both? : Both (Server Multiplayer, issue happens to Players)

**Comment by happybandit360** [comment]
This is the nature of this mod as the developer intended when it was made. We added a mod that lets you create a Rifting Machine to let you enter and leave at your convenience. Waystones and Signposts are also helpful.

**Comment by JoelZBub** [comment]
One thing to keep in mind is it needs to be nighttime for them to appear in Midnight. If server players are constantly sleeping the night away and making it daytime, you'll never see one. Keep an eye on the clock in your mini map.

**Comment by Vaygrim** [comment]
Wait, does that mean the exit rifts ARE supposed to spawn? Or are they not?
I'm getting two different messages here.

**Comment by Vaygrim** [comment]
Also, second question: If I setup a brand new "Uncle Genny's 100% Organic" server and have players abducted on the FIRST NIGHT, and Rifts are NOT supposed to be showing up to get out Midnight... what the heck are they supposed to do?

**Comment by happybandit360** [comment]
Rifts DO spawn in the Midnight, but only during night in the Overworld. You just have to look for them

**Comment by JoelZBub** [comment]
We've given you all the information you need to be successful. Please go back and re-read our reply

**Comment by Vaygrim** [comment]
I stated that we are NOT seeing exit rifts in Midnight when it is night time in the Overworld.
First response I get from "happybandit360" is: `"This is the nature of this mod as the developer intended when it was made. We added a mod that lets you create a Rifting Machine to let you enter and leave at your convenience. Waystones and Signposts are also helpful."`
No indication that the rifts not showing was a bug, or "Huh, that's weird. Screenshots?". No you told me it was working as intended by the developer. This means the rifts aren't SUPPOSED to be spawning. Next comment from "JoelZBub" then directly contradicts that.

I'm not trying to make trouble, honest. I'm just trying to figure out why we keep getting solo players trapped in Midnight for hours on end when no one else is on the server. Is this an issue that's beyond the scope of support for this pack? If so just tell me, I'll go ask the mod dev.
